pandas_gbq/gbq.py

```diff
@@ -767,6 +770,116 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
     return final_df
 
+def from_gbq(query, project_id=None, index_col=None, col_order=None,
```

Comment:
Sure, I can rename mine to `read_gbq` and add in the query length/bytes processed info. Want me to delete the old `read_gbq` and related code or just rename it?